src/core/highlight.js

```diff
@@ -452,10 +452,10 @@ class Highlight {
      * @example
      * const highlight = esb.highlight('content').type('plain', 'content');
      *
-     * @param {string} type The allowed values are: `plain`, `postings` and `fvh`.
+     * @param {string} type The allowed values are: `plain`, `postings`, `unified` and `fvh`.
```

Comment:
Would it be possible to add a note regarding the Elasticsearch version from which `unified` type is supported?

Reply:
added. I tracked down exactly the alpha when they released it: https://github.com/elastic/elasticsearch/pull/21621